android/TPSDropDown/testmodule/src/test/java/com/gettipsi/testmodule/ExampleUnitTest.java

```diff
@@ -0,0 +1,15 @@
+package com.gettipsi.testmodule;
```

Comment:
Still don't understand what is for, it doesn't test anything useful:
`
  public class ExampleUnitTest {

     @Test
     public void addition_isCorrect() throws Exception {
         assertEquals(4, 2 + 2);
     }
  } 

`